README.md

```diff
@@ -57,3 +57,12 @@
 ### How to use with ZSH
 * put `export KUBECONFIG=[path-to-repo]/content-k8s-auth-setup/kubeconfig`  in `~/.zshrc`
 * execute `source cluster-login.zsh udde`
+
+
+### Only if you are connecting to EKS
+When provisioning a new EKS cluster the provisioning script will upload the corresponding kubeconfig to S3 bucket named "upp-kubeconfig". In order to use it you need to get this kubeconifg and merge it with the kubeconfig files of the rest of EKS clusters. This way you will have a single united kubeconfig for all the EKS clusters. Inside `update-eks-kubeconfig` directory there is a script named `update-eks-kubeconfig.sh`. The script will do this merge for you and will store the merged kubecofig in /tmp directory of your local computer:
```

Comment:
- It would be nice to tip the reader to create an alias to change the KUBECONFIG. For example:
```
# EKS
alias export-eks-kubeconfig='export KUBECONFIG=/path/to/file/eks-kubeconfig.yaml'
```

Reply:
* I don't intend here to use the generated kubeconfig from within the /tmp directory. It is up to the  user to move it to where he/she wants to keep it and export KUBECONFIG from there.

Reply:
Shortened the text

Reply:
I have rewrapped with line width 100